Documentation/Ctrl/Properties/PreviewRenderer.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: /Includes.rst.txt
+.. _ctrl-reference-previewrenderer:
+
+===============
+previewRenderer
+===============
+
+.. confval:: previewRenderer
+
+   :Path: $GLOBALS['TCA'][$table]['ctrl']
+   :type: string
+   :Scope: Display
+
+   Configure a backend preview for a content element.
+
+Examples
+========
+
+Have also a look at :ref:`t3coreapi:ConfigureCE-Preview` for more details.
+
+Use for any record in a table
+-----------------------------
+
+This specifies the preview renderer to be used for any record in :sql:`my_table`:
+
+..  code-block:: php
+
```

Comment:
Where would we put these examples? In the overrides? We should state so

Reply:
The example as is works only in overrides, but you can also configure in the default table. So I stepped away from adding a caption.

Reply:
I added a note.